addons/l10n_id_efaktur/views/account_move_views.xml

```diff
@@ -6,19 +6,19 @@
             <field name="model">account.move</field>
             <field name="inherit_id" ref="account.view_move_form"/>
             <field name="arch" type="xml">
-                <field name="partner_id" position="after">
+                <field name="partner_shipping_id" position="before">
                     <field name="l10n_id_need_kode_transaksi" invisible="1"/>
-                    <field name="l10n_id_attachment_id" invisible="1"/>
                     <field name="l10n_id_kode_transaksi" invisible="country_code != 'ID' or not l10n_id_need_kode_transaksi" readonly="state in ['cancel', 'posted']" required="l10n_id_need_kode_transaksi"/>
-                    <field name="l10n_id_replace_invoice_id" invisible="country_code != 'ID'" readonly="state != 'draft'" options="{'m2o_dialog': False, 'no_create': True}"/>
+                    <field name="l10n_id_efaktur_range" invisible="country_code != 'ID' or state != 'draft' or l10n_id_available_range_count &lt; 2" domain="[('available', '>', 0)]"/>
```

Comment:
Please et the domain in the field declaration.
Also, I think you can add `options="{'no_create': True}"` here, it wouldn't make sense to create a range from there